microcosm_pubsub/matchers/message.py

```diff
@@ -76,7 +76,7 @@ def iter_from_sns_producer(sns_producer) -> Iterable["PublishedMessage"]:
         Iterate over published messages from a mocked SNSProducer.
 
         """
-        if isinstance(sns_producer, Mock):
+        if isinstance(sns_producer, Mock):  # type: ignore
```

Comment:
This was raising a new error (`Parameterized generics cannot be used with class or instance checks`) (this is also on develop, btw).